layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -185,3 +189,7 @@ do_deploy_append_srd3-tx2() {
 do_deploy_append_blackboard-tx2() {
     cp ${WORKDIR}/tegra186-tx2-blackboard.dtb "${DEPLOYDIR}"
 }
+
+do_deploy_append_jn30b-nano() {
```

Comment:
As now Jetson Nano splits into two separate devices,  it's necessary to use different dtbs for tegra210-flash too, like we did here for the tx2: https://github.com/balena-os/balena-jetson/commit/fce0477aa41c74c2698a56077b15bfb9e563e609

Reply:
Made the split within tegra210-flash by including KERNEL_DEVICETREE_jn30b-nano